temba/utils/gsm7.py

```diff
@@ -377,220 +380,30 @@ def calculate_num_segments(text):
-    "ʻ": "'",
-    "ˈ": "'",
-    "ʼ": "'",
-    "ʽ": "'",
-    "ʹ": "'",
-    "‛": "'",
-    "́": "'",
-    "ˊ": "'",
-    "ˋ": "'",
-    "❛": "'",
-    "❜": "'",
-    "̓": "'",
-    "̔": "'",
-    "、": ",",
-    "÷": "/",
-    "1⁄4": "1/4",
-    "1⁄2": "1/2",
-    "3⁄4": "3/4",
-    "⧸": "/",
-    "̷": "/",
-    "̸": "/",
-    "⁄": "/",
-    "∕": "/",
-    "⧹": "\\",
-    "⧵": "\\",
-    "": "",
-    "̲": "_",
-    "⃒": "|",
-    "⃓": "|",
-    "∣": "|",
-    "⎸": "|",
-    "⎹": "|",
-    "⏐": "|",
-    "⎜": "|",
-    "⎟": "|",
-    "⎼": "-",
-    "⎽": "-",
-    "―": "-",
-    "‐": "-",
-    "•": "-",
-    "⁃": "-",
-    "ǃ": "!",
-    "‚": "'",
-    "̦": ",",
-    "❨": "(",
-    "❪": "(",
-    "⟮": "(",
-    "⦅": "(",
-    "❩": ")",
-    "❫": ")",
-    "⟯": ")",
-    "⦆": ")",
-    "⁎": "*",
-    "∗": "*",
-    "⊛": "*",
-    "✢": "*",
-    "✣": "*",
-    "✤": "*",
-    "✥": "*",
-    "✱": "*",
-    "✲": "*",
-    "✳": "*",
-    "✺": "*",
-    "✻": "*",
-    "✼": "*",
-    "✽": "*",
-    "❃": "*",
-    "❉": "*",
-    "❊": "*",
-    "❋": "*",
-    "⧆": "*",
-    "˖": "+",
-    "。": ".",
-    "ː": ":",
-    "˸": ":",
-    "⦂": ":",
-    "꞉": ":",
-    "⁏": ";",
-    "͇": "=",
-    "꞊": "=",
-    "ᴀ": "A",
-    "ʙ": "B",
-    "ᴄ": "C",
-    "ᴅ": "D",
-    "ᴇ": "E",
-    "ꜰ": "F",
-    "ɢ": "G",
-    "ʜ": "H",
-    "ɪ": "I",
-    "ᴊ": "J",
-    "ᴋ": "K",
-    "ʟ": "L",
-    "ᴍ": "M",
-    "ɴ": "N",
-    "ᴏ": "O",
-    "ᴘ": "P",
-    "ʀ": "R",
-    "ꜱ": "S",
-    "ᴛ": "T",
-    "ᴜ": "U",
-    "ᴠ": "V",
-    "ᴡ": "W",
-    "ʏ": "Y",
-    "ᴢ": "Z",
-    "ˆ": "^",
-    "̂": "^",
-    "᷍": "^",
-    "❴": "{",
-    "❵": "}",
-    "̃": "~",
-    "˷": "~",
-    "̰": "~",
-    "̴": "~",
-    "∼": "~",
-    "​": "",
-    "̳": "_",
-    "—": "-",
-    "–": "-",
-    "‹": ">",
-    "›": "<",
-    "!!": "!!",
-    "„": '"',
-    "...": "...",
-    "⁠": "",
-    "\t": " ",
-    "\xa0": " ",
-    "¡": "",
-    "£": "",
-    "¤": "",
-    "¥": "",
-    "§": "",
-    "¿": "",
-    "À": "A",
-    "Á": "A",
-    "Â": "A",
-    "Ã": "A",
-    "Ä": "A",
-    "Å": "A",
-    "Æ": "E",
-    "Ç": "C",
-    "È": "E",
-    "É": "E",
-    "Ê": "E",
-    "Ì": "I",
-    "Í": "I",
-    "Î": "I",
-    "Ñ": "N",
-    "Ò": "O",
-    "Ó": "O",
-    "Ô": "O",
-    "Õ": "O",
-    "Ö": "O",
-    "Ø": "O",
-    "Ù": "U",
-    "Ú": "U",
-    "Û": "U",
-    "Ü": "U",
-    "ß": "",
-    "à": "a",
-    "â": "a",
-    "ã": "a",
-    "ä": "a",
-    "å": "a",
-    "æ": "e",
-    "ç": "c",
-    "è": "e",
-    "é": "e",
-    "ê": "e",
-    "ì": "i",
-    "í": "i",
-    "î": "i",
-    "ñ": "n",
-    "ò": "o",
-    "ó": "o",
-    "ô": "o",
-    "õ": "o",
-    "ö": "o",
-    "ø": "o",
-    "ù": "u",
-    "ú": "u",
-    "û": "u",
-    "ü": "u",
-    "Δ": "",
-    "Θ": "O",
-    "Λ": "",
-    "Ξ": "",
-    "Π": "",
-    "Σ": "",
-    "Φ": "",
-    "Ψ": "",
-    "Ω": "",
-}
+def load_gsm7_replacements():
+    handle = open("%s/flow_editor/gsm7-replacements.json" % settings.MEDIA_ROOT, "r+")
```

Comment:
@musamusa, are you able to access the gsm7-replacements.json from the floweditor library? I think it would be nice to have only one file with replacements so the behavior would not be different if one file changed and another is not.

Reply:
I thought of that, let me try if I can from the `node_modules` folder

Reply:
nope, it is not accessible as a file, it gets compiled into the minified js output.